airbyte-webapp/src/components/ui/Card/Card.tsx

```diff
@@ -31,14 +35,22 @@ export const Card: React.FC<React.PropsWithChildren<CardProps>> = ({
       })}
     >
       {title ? (
-        <H5
-          className={classNames(styles.title, {
+        <div
+          className={classNames(styles.header, {
             [styles.lightPadding]: lightPadding || !children,
             [styles.roundedBottom]: roundedBottom,
           })}
         >
-          {title}
-        </H5>
+          <H5 className={classNames(styles.title)}>{title}</H5>
+          {description ? (
```

Comment:
Adding description of what a Source/Destination is in a tooltip since we are loosing the message coming from the empty state